db/migrate/20170929131411_add_fields_to_polls.rb

```diff
@@ -0,0 +1,7 @@
+class AddFieldsToPolls < ActiveRecord::Migration
+  def change
+    add_column :polls, :comments_count, :integer, default: 0
+    add_column :polls, :hidden_at, :datetime
```

Comment:
No need to hide polls

Reply:
Better to remove de migration file from the PR (and revert the db/schema.rb) than adding a migration that reverts the changes 🙏

Reply:
@bertocq the whole migration? comments_count is not needed elsewhere?

Reply:
Right, not the entire migration, seems like `comments_count` its needed indeed 👍

Reply:
After running a migration that removes both hidden_at and author_id, it raises an error:
```
PG::UndefinedColumn: ERROR: no existe la columna polls.hidden_at LINE 1: SELECT "polls".* FROM "polls" WHERE "polls"."hidden_at" IS ... ^ : SELECT "polls".* FROM "polls" WHERE "polls"."hidden_at" IS NULL AND "polls"."id" = $1 LIMIT 1
```